contracts/RewardController.vy

```diff
@@ -27,6 +27,9 @@ event OracleUpdated:
     deviation_reward: uint256
     reward_mult: int256
 
+event RewardsStatus:
```

Comment:
Its about `event OracleUpdated:` right above. 
So if you allow OracleUpdated when it's frozen, shouldn't if have an explicit flag that it's not during the game? If not a flag, maybe different event like FrozenUpdate with only few fields to distinct?

Also I think we should populate types in events  (and pretty much everywhere) not a huge costs to us but seems like something we would really regret if we wouldn't

Reply:
OracleUpdated is impossible when frozen(as everything reverts), but possible when rewards are disabled.
When rewards are disabled, both rewards will be 0. That's the sign that its not during the game.